ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallViewModelFactory.kt

```diff
@@ -0,0 +1,11 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.revenuecat.purchases.Offering
+
+class PaywallViewModelFactory(private val offering: Offering?) : ViewModelProvider.NewInstanceFactory() {
```

Comment:
To pass parameters to the view model, a factory like this is required... One of the problems of architecture component's viewModel.